data/json/mapgen/musicstore.json

```diff
@@ -38,7 +38,7 @@
           "s_music"
         ],
         "method": "json",
-        "weight": 180, "comment": "<-- Real decent place for a quick morale boost, relatively rare.",
+        "weight": 50, "comment": "<-- Real decent place for a quick morale boost, relatively rare.",
```

Comment:
Aha! You are correct!

Mapgen's still a bit of a mystery to me, as it's the only part of the code I've not really played around in yet.